src/styles/What.css

```diff
@@ -53,18 +53,34 @@
   width: 100px;
   height: 100px;
   padding: 0px;
-  border-radius: 80px;
-  box-shadow: 5px 5px 5px grey;
-  margin-right: 40px;
-
+  margin-top: auto;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: auto;
 }
 
 .RightBlurbImages {
   float: left;
   width: 100px;
   height: 100px;
   padding: 0px;
-  box-shadow: 5px 5px 5px grey;
-  border-radius: 80px;
+  margin-top: auto;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: auto;
+}
+
+#wwdLeftBlurb {
+  padding: 0px;
+  border-radius: 50%;
+  margin-right: 40px;
+  box-shadow: 5px 5px 5px grey
+}
+
+#wwdRightBlurb {
+  width: 110px;
+  height: 110px;
+  border-radius: 50%;
   margin-left: 40px;
+  box-shadow: 5px 5px 5px grey;
```

Comment:
What.css: Code is clean. Good tweaking of class and id attributes to reflect element styling.